### PR TITLE
Harmonize way to provide intl

### LINKED ISSUE #193: There are 2 ways how Intl is provided to forms
First way:
```
  const configurationContext = useContext(ConfigurationContext);

  const intl = configurationContext.options.intl;
```

Second way:
```
IntlContextProvider
```

There should be only one way -- i.e it should be unified.

### PR DESCRIPTION
**Pull request by LaChope**
Resolves #193.

Sorry this is quite big PR in the end but it was necessary to really harmonize how we provide intl.

### CODE REVIEW

File: src/i18n/cs.json [hunk 1]
```diff
@@ -1,4 +1,16 @@
 {
   "comment.form.placeholder": "Sem napište své komentáře (Ctrl+Enter pro potvrzení)",
-  "wizard.nav.tooltip": "Navigace na této kartě je zakázána, aby nedošlo k přehlédnutí důležitých otázek. K navigaci na tuto kartu použijte tlačítka související s formulářem."
+
```
Comment (LaChope) [inline src/i18n/cs.json:3]: I translated all this part with Deepl, it needs to be revised

**Review by blcham — APPROVED** [review]
LGTM, it is hard to check, can it break s-forms-components?

**Comment by blcham** [comment]
@LaChope you can merge, just answer me last question :)

**Comment by LaChope** [comment]
@blcham It will not break s-forms-components, but it can be refactored there as <IntlContextProvider> wrapping the example story is n longer necessary.

**Pull request merged**